core/startup/installers.go

```diff
@@ -248,9 +248,16 @@ func installIdeFromZip(archivePath string, targetDir string) error {
 	}
 
 	err = os.Rename(tempDir, targetDir)
-	if err != nil {
-		return fmt.Errorf("error moving linter files from temp to target: %w", err)
+	if err != nil { // Trying to fallback if rename failed https://youtrack.jetbrains.com/issue/QD-12252
+		log.Warningf("Error moving linter files from temp to target: %s. Trying to copy instead.", err)
+		_, err := os.Stat(targetDir)
+		if err == nil || os.IsNotExist(err) {
```

Comment:
`os.IsNotExist` help tells me we should use `errors.Is(err, fs.ErrNotExist)` in new code

Reply:
Also the "not exist" condition should be inverted